portal/import_template.php

```diff
@@ -293,7 +293,13 @@ function renderProfileHtml()
     <!DOCTYPE html>
     <html>
     <head>
-        <?php Header::setupHeader(['opener', 'sortablejs']); ?>
+        <?php
+        if (empty($GLOBALS['openemr_version'] ?? null)) {
```

Comment:
is this if block meant for rel-601 or master?

Reply:
both for now. I want to be able to move portal between versions and not worry with custom to any one version.